Dockerfile

```diff
@@ -110,15 +119,20 @@ RUN chown www-data:www-data /var/www/html/w/images
 RUN chmod 777 /var/www/html/w/cache
 COPY mardi_php.ini /usr/local/etc/php/conf.d/mardi_php.ini 
 
-# Set up vecollabpad
-# RUN cd /var/www/html/w/extensions/VisualEditor/lib/ve && npm install && grunt build
-# RUN cd /var/www/html/w/extensions/VisualEditor/lib/ve/rebaser && npm install && cp config.dev.yaml config.yaml && sed -i 's/localhost/mongodb/g' config.yaml
-
 ENV TZ=Europe/Berlin
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 RUN printf '[PHP]\ndate.timezone = "Europe/Berlin"\n' > /usr/local/etc/php/conf.d/tzone.ini
 
-##
+# Set Apache environment variables
+ENV APACHE_RUN_USER=www-data \
+    APACHE_RUN_GROUP=www-data \
+    APACHE_LOG_DIR=/var/log/apache2 \
+    APACHE_PID_FILE=/var/run/apache2/apache2.pid \
+    APACHE_RUN_DIR=/var/run/apache2 \
+    APACHE_LOCK_DIR=/var/lock/apache2
+
+RUN mkdir -p $APACHE_RUN_DIR $APACHE_LOCK_DIR $APACHE_LOG_DIR
```

Comment:
Do we need this at all, I don't see it here

https://github.com/wikimedia/mediawiki-docker/blob/22e0f2939d36bfc61d1572e4e1a2afd66d84e6f5/1.44/fpm-alpine/Dockerfile#L93

The only thing I am a bit worried about is the question where the logs should go, but other than that I think our setup is quite standard.